operating/respond-to-violations.adoc

```diff
@@ -0,0 +1,42 @@
+[id="respond-to-violations"]
+= Responding to violations
+include::modules/common-attributes.adoc[]
+:context: respond-to-violations
+
+toc::[]
+
+[role="_abstract"]
+Using {product-title} you can view policy violations, drill down to the actual cause of the violation, and take corrective actions.
+
+{product-title} built-in policies identify a variety of security findings, including vulnerabilities (CVEs), violations of DevOps best practices, high-risk build and deployment practices, and suspicious runtime behaviors.
+Whether you use the default out-of-box security policies or use your own custom policies, {product-title} reports a violation when an enabled policy fails.
+
+include::modules/violations-view.adoc[leveloffset=+1]
+
+[id="view-violation-details"]
+== Viewing violation details
+When you select a violation in the *Violations* view, the *Violation Details* open in a panel on the right.
+
+The *Violation Details* panel shows detailed information grouped by multiple tabs.
+
+include::modules/violation-view-violation-tab.adoc[leveloffset=+2]
+
+[id="using-comments-and-tags"]
+=== Using comments and tags
```

Comment:
Just checking that this is the right spot for this. Looking at the TOC, it seemed out of place with all of the other sections being a particular "tab". Is this particular to the violations tab and should be indented?